blocklist.csv

```diff
@@ -162,21 +189,24 @@
 2886611010,OverwatchES,Overwatch
 2886598926,OverwatchDE,Overwatch
 2875960369,WagWalking,Wag! - The #1 App for Pet Parents
+2874959663,Russia,Russia 🇷🇺
 2874803581,steamintech,Coffee + Tech
 2873526639,NOPLPoff,NOPLP Officiel
 2865691205,365ninja,365 Ninja
 2857454552,Uber_TX,Uber Texas
 2857406022,uber_phx,Uber Phoenix
 2848423391,ITI_Jobs,Infinite Technologie
 2834176185,WinamaxSport,Winamax Sport 🔞
-2832518219,NRM_inc,|NRM, Inc.|
+2832518219,NRM_inc,|NRM
```

Comment:
The ,Inc.| part seems to have been moved somewhere else, the CSV won't be valid without the matching "|"